src/detect-sip-stat-code.c

```diff
@@ -56,7 +56,7 @@
 
 #define KEYWORD_NAME "sip.stat_code"
 #define KEYWORD_DOC  "sip-keywords.html#sip-stat-code"
-#define BUFFER_NAME  "sip.method"
+#define BUFFER_NAME  "sip.stat_code"
```

Comment:
Thanks for this.

Question for the team: can this lead to wrong detection when using `sip.stat_code`and `sip.method` keywords ?

Note: this was fixed in master by moving the code to rust

Reply:
> Question for the team: can this lead to wrong detection when using `sip.stat_code`and `sip.method` keywords ?

Don't know anything about `sip` but we do have s-v tests for this which pass both in `master` and this PR:
1. `sip-stat-code` which passes. I can confirm that it passes correctly. Wireshark filter `sip.Status-Code==100` shows exactly the number of matches expected.
2. `sip-method` which passes. I can confirm that this passes correctly too. Wireshark filter `sip.Method==REGISTER` shows exactly the number of matches expected.

I haven't tried to look how the `BUFFER_NAME` impacts the parsing and storage of the respective fields..